google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -190,7 +190,6 @@ public GapicSpannerRpc(SpannerOptions options) {
                         options.getInterceptorProvider(),
                         SpannerInterceptorProvider.createDefault()))
                 .setHeaderProvider(mergedHeaderProvider)
-                .setExecutorProvider(InstantiatingExecutorProvider.newBuilder().build())
```

Comment:
Since each underlying grpc stub gets set this executor provider by default, is it correct that each stub will not use a shared executor for its transport channel (the one potential problem listed in the issue with this change)?

Reply:
Correct.